rust-toolchain.toml

```diff
@@ -1,3 +1,3 @@
 [toolchain]
-channel = "1.70.0"
+channel = "stable"
```

Comment:
Do not make this change. The `rust-toolchain.toml` file is what enforces MSRV in CI tests, and ensures that developers don't accidentally rely on newer Rust features while making local changes to the crate.